flink-filesystems/flink-s3-fs-presto/pom.xml

```diff
@@ -33,7 +33,7 @@ under the License.
 	<packaging>jar</packaging>
 
 	<properties>
-		<presto.version>0.187</presto.version>
+		<presto.version>0.257</presto.version>
```

Comment:
We didn't technically test which of the two changes fixed it (since the testing roundtrip takes so long), but I suspect it it's the presto-hive fix, not the bump to the AWS dependencies. The AWS dependencies were already on a version that should work (and by quite a bit), but the difference between the two presto-hive versions is that the sts dependency is now also in presto-hive when it wasn't before.

s3-fs-hadoop isn't fixed by any of these changes. For Hadoop you need special configuration because Hadoop ships its own credentials provider chain mechanism.